tests/e2e/lib/container

```diff
@@ -121,11 +118,11 @@ create_qm_node() {
 
         # create the container ${nodeID}
         info_message "Creating container \033[92mnode${nodeID}\033[0m [\033[92mQM mode\033[0m]"
+        CONTAINER_ADD_CAPS="SYS_ADMIN /tmp/"
         create_container \
                 ContainerFile.node"${nodeID}" \
                 "node${nodeID}" \
-                "node:latest" \
-                "SYS_ADMIN /tmp/"
+                "node:latest"
```

Comment:
shouldn't use the CONTAINER_ADD_CAPS here?

Reply:
ACK